Bugsnag/Client/BugsnagClient+Private.h

```diff
@@ -60,7 +60,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (strong, nonatomic) BugsnagSessionTracker *sessionTracker; // Used in BugsnagReactNative
 
-@property (readonly, nonatomic) BOOL started;
```

Comment:
I get that it mutes the warning, but is it an improvement to make this property (and others) mutable to the rest of the library to avoid ivar access?

Reply:
In this case, maybe not; setting it to NO would be very bad.

Reply:
Resolved 👍